internal/display/apis.go

```diff
@@ -112,12 +112,12 @@ func (r *Renderer) ApiUpdate(api *management.ResourceServer) {
 func makeApiView(api *management.ResourceServer) (*apiView, bool) {
 	scopes, scopesTruncated := getScopes(api.Scopes)
 	view := &apiView{
-		ID:            auth0.StringValue(api.ID),
-		Name:          auth0.StringValue(api.Name),
-		Identifier:    auth0.StringValue(api.Identifier),
-		Scopes:        auth0.StringValue(scopes),
-		TokenLifetime: auth0.IntValue(api.TokenLifetime),
-		OfflineAccess: auth0.BoolValue(api.AllowOfflineAccess),
+		ID:            ansi.Faint(api.GetID()),
```

Comment:
just a personal preference: I'd like to avoid multiple command changes that are unrelated. As trivial as those can be, smaller PRs are easier to review, revert (if needed 🤞🏼 ), trace old changes if needed (troubleshooting).

Reply:
Yes, that's true. However, it's far easier to get a reviewer for a single PR. However, I'll try to split these into their own PRs in the future.

Reply:
thanks 🙇🏼

Reply:
I believe smaller PRs would also lower the barrier for new reviewers, give feedback on a ~50 files PR can be overwhelming 😅